frontend/src/metabase/dashboard/actions/parameters.unit.spec.js

```diff
@@ -0,0 +1,32 @@
+import { removeParameter, REMOVE_PARAMETER } from "./parameters";
+
+describe("removeParameter", () => {
+  let dispatch;
+  let getState;
+  beforeEach(() => {
+    dispatch = jest.fn();
+    getState = () => ({
```

Comment:
shouldn't we prepare real type safe state? @kamilmielnik

Reply:
Ideally, yes.
But I don't think we have a single unit test with real type safe state yet, do we?

BTW this test uses state definition almost identical to another one from existing test for action creators: https://github.com/metabase/metabase/blob/8fb7a7842528197567c523ae921b79ce055d3bf9/frontend/src/metabase/dashboard/actions/actions.unit.spec.js#L26-L36